.bazelrc

```diff
@@ -15,16 +15,15 @@ build:clang-cl --compiler=clang-cl
 build:msvc --compiler=msvc-cl
 # `LC_ALL` and `LANG` is needed for cpp worker tests, because they will call "ray start".
 # If we don't add them, python's `click` library will raise an error.
-test --action_env=LC_ALL
-test --action_env=LANG
+build --action_env=LC_ALL
+build --action_env=LANG
 # Allow C++ worker tests to execute "ray start" with the correct version of Python.
-test --action_env=VIRTUAL_ENV
-test --action_env=PYENV_VIRTUAL_ENV
-test --action_env=PYENV_VERSION
-test --action_env=PYENV_SHELL
-test --action_env=RAY_ENABLE_NEW_SCHEDULER
+build --action_env=VIRTUAL_ENV
+build --action_env=PYENV_VIRTUAL_ENV
+build --action_env=PYENV_VERSION
+build --action_env=PYENV_SHELL
 # This is needed for some core tests to run correctly
-test:windows --enable_runfiles
+build:windows --enable_runfiles
```

Comment:
I'm assuming this is fine for Windows builds, if not I can set this back to `test:windows`.